apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -115,16 +115,23 @@ function DescriptionSection(props: DescriptionProps) {
   );
 }
 
-type CtaProps = Pick<Twc2SupportDetail, 'ctaBlurb' | 'ctaButtonText'>;
+type CtaProps = Pick<
+  Twc2SupportDetail,
+  'ctaBlurb' | 'ctaButtonText' | 'ctaLink'
+>;
 
 function CtaSection(props: CtaProps) {
-  const { ctaBlurb, ctaButtonText } = props;
+  const { ctaBlurb, ctaButtonText, ctaLink } = props;
   return (
     <Box className="support-cta-container" sx={{ borderTopColor: 'grey.200' }}>
       <div style={{ flex: 4, height: '100%' }}>{ctaBlurb}</div>
       <PrimaryButton
+        href={ctaLink}
+        target="_blank"
+        rel="noopener"
         sx={{ flex: 5, backgroundColor: 'primary.700', height: '100%' }}
```

Comment:
@e056 should we move this into theme? CC: @glencbz

Reply:
Hi @stevahnes , sorry for the late reply. Sorry which part are you referring to?

Reply:
The `sx` portion

Reply:
I don't think you meant theme? The `sx` is already referencing theme variables (e.g. `primary.700`).

Perhaps you meant making this a shareable component. That will be good to do someday, but that's a refactor that's out of scope for this feature.

Reply:
Hmmm, I initially thought that we should probably define a theme for all primary buttons instead? Unless we foresee a different primary button look and feel?

Reply:
Hm, I don't follow. The word 'theme' sounds like you're referring to Material UI themes (https://mui.com/material-ui/customization/theming/), but that can't be what you mean.

It sounds like you mean custom styles and a custom component (which is what I described). In which case, yes we do want a fixed set of styles for our CTA button. But @e056 isn't adding a new instance of the button or making any substantial change that warrants us refactoring the code yet, so I don't think that needs to be done noow.